### PR TITLE
Throw an error for any item that resolves as undefined

### PR DESCRIPTION
**Pull request by nturinski**
Mitigates https://github.com/microsoft/vscode-azurefunctions/issues/3851 and other issues, I'm sure

We see this error floating around quite often. While this doesn't actually fix it, at least the error makes a little more sense to the user.

### CODE REVIEW

**Review by bwateratmsft — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -571,7 +571,7 @@
     },
     "scripts": {
         "vscode:prepublish": "npm run cleanReadme",
-        "build": "tsc && gulp webpack-prod",
+        "build": "tsc",
```
Comment (bwateratmsft) [inline package.json:574]: Is this intentionally included?

Comment (nturinski) [reply package.json:574]: Yeah, the `npm run build` shouldn't run webpack. It wasn't actually changing the line `return await extension.activate(ctx, perfStats, true);` either (where the true is ignoreBundle) so it wasn't actually prepping it to be used with webpack.

Comment (alexweininger) [reply package.json:574]: Can confirm that this matches our other extensions

**Review by bwateratmsft — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (nturinski) [reply package.json:574]: Yeah, agreed: https://github.com/microsoft/vscode-azureresourcegroups/issues/826

**Review by alexweininger — DISMISSED** [review]

**Review by alexweininger — APPROVED** [review]

**Pull request merged**